docs/serverless-v2/08-03-bind-service-instance.md

```diff
@@ -145,6 +151,57 @@ Follows these steps:
 
     > **NOTE:** If you added the **REDIS_** prefix for environmental variables in step 6, all variables will start with it. For example, the **PORT** variable will take the form of **REDIS_PORT**.
 
+    </details>
+    <details>
+    <summary label="console-ui">
+    Console UI
+    </summary>
+
+> **NOTE:** Serverless v2 is an experimental feature, and it is not enabled by default in the Console UI. To use its **Functions [preview]** view, enable **Experimental functionalities** in the **General Settings** view before you follow the steps.
+
+To create a binding, you must first create a sample service instance to which you can bind the lambda. Follow the sections and steps to complete this tutorial.
+
+### Provision a Redis service using an Addon
+
+> **NOTE:** If you already have a Redis instance provisioned on your cluster, move directly to the **Bind the lambda with the service Instance** section.
+
+Follow these steps:
+
+1. Select a Namespace from the drop-down list in the top navigation panel where you want to provision the Redis service.
+2. Go to the **Addons** view in the left navigation panel and select **Add New Configuration**.
+3. Enter `https://github.com/kyma-project/addons/releases/download/0.11.0/index-testing.yaml` in the **Urls** field. The Addon name is automatically generated.
+4. Select **Add** to confirm changes.
+
+    You will see that the Addon has the `Ready` status.
+
+### Create a Service Instance
+
+1. Go to the Catalog view where you can see the list of all available Addons and select **Redis**.
```

Comment:
```suggestion
1. Go to the **Catalog** view where you can see the list of all available Addons and select **[Experimental] Redis**.
```
But it may also be Redis - it's up to you. Also, the Console itself doesn't have the names unified (Addons vs Add-Ons), but that's another story :)

Reply:
Sure, I will write **[Experimental] Redis** here as this is a specific example based on this specific Addon.